spec/controllers/tag_wranglers_controller_spec.rb

```diff
@@ -17,4 +17,61 @@
       end
     end
   end
+
+  describe "#report_csv" do
+    let!(:tag1) { create(:freeform, last_wrangler: user, unwrangleable: true) }
+    let!(:tag2) { create(:relationship, last_wrangler: user) }
+
+    context "when anonymous" do
+      it "prevents access" do
+        get :report_csv, params: { id: user.login }
+        it_redirects_to_simple(root_path)
+      end
+    end
+
+    context "when logged in as a superadmin" do
```

Comment:
Can we test that admins with other roles (including no roles) can't access the report? Usually we just loop through the roles like in [the skins controller specs](https://github.com/otwcode/otwarchive/blob/dc8fac616e4841f5bc600327ea7cc1a75ee48d89/spec/controllers/skins_controller_spec.rb#L28-L48).

Reply:
Done!